packages/components/inputs/SearchNSInputContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from "react";
+import { StyleProp, View, ViewStyle } from "react-native";
+
+import { useNameSearch } from "../../hooks/search/useNameSearch";
+import { useSelectedNetworkId } from "../../hooks/useSelectedNetwork";
+import { parseUserId } from "../../networks";
+import { layout } from "../../utils/style/layout";
+import { AvatarWithName } from "../user/AvatarWithName";
+
+// Used as a wrapper on a TextInput or something like. It allows to search a name in TNS and returns the address.
+export const SearchNSInputContainer: FC<{
+  searchText: string;
+  onPressName: (address: string) => void;
```

Comment:
better to pass user id here to support multi-chain (we have gno names support in another PR for example)